### PR TITLE
Refactoring awesome books

### PR DESCRIPTION
**Pull request by David-Lanzz**
1. We have refactored the codes to use object constructors
2. We have checked for linter errors

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -55,4 +55,4 @@ let objVar = '';
 for (let i = 0; i < bookObj.length; i += 1) {
   objVar += `<li><p><span>${bookObj[i].bookTitle}</span> by <span>${bookObj[i].arthur}</span> </p> <button id=${bookObj[i].id.toString()} onClick="removeBook(${bookObj[i].id})">Remove</button></li>`;
 }
-bookList.innerHTML = objVar;
+bookList.innerHTML = objVar;
```
Comment (danmainah) [inline js/main.js]: - [ ] @David-Lanzz and @stephenkati great work on your application, however, you forgot to implement the classes for add and delete functions, also ensure your arrays are inside classes this is a project requirement
image

File: index.html [hunk 2]
```diff
@@ -52,7 +52,6 @@ <h3>Contact Us</h3>
             copyright
         </footer>
```
Comment (danmainah) [inline index.html:53]: - [ ] [optional] It would be nice if the footer position was not fixed so that when you add new books the design doesn't get broken

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @David-Lanzz and @stephenkati , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] @team kindly sure your license link is working, currently it's broken.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Owusu-Desmond — APPROVED** [review]
Hi @David-Lanzz and @stephenkati , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**